guides/app-developers/es-usage-guide.md

```diff
@@ -0,0 +1 @@
+Follow the tutorial [here](https://docs.ethstorage.io/dapp-developer-guide).
```

Comment:
This link (https://docs.ethstorage.io/dapp-developer-guide/tutorials) is more straight forward for developers

Reply:
Done in eb1db9a34f18f96fc9e2e0ddbeed241b3922f768.